spring-cloud-starter-single-step-batch-job/src/main/java/org/springframework/cloud/task/batch/autoconfigure/rabbit/AmqpItemWriterProperties.java

```diff
@@ -26,8 +26,15 @@
 @ConfigurationProperties(prefix = "spring.batch.job.amqpitemwriter")
 public class AmqpItemWriterProperties {
 
+	/**
+	 * Enables or disables the AmqpItemWriter.  Defaults to false.
```

Comment:
Change the double space to a single space and make "false" be code: {@code false}